live-site/components/mentors-section/MentorsSection.styles.tsx

```diff
@@ -142,7 +146,15 @@ const StyledMentorsPaginationContainer = styled.div`
   @media ${min.desktop} {
     max-width: 95%;
   }
-`
+`;
+
+const StyledNoMentorsContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 8px;
```

Comment:
use em SORRY

Reply:
lmfao i have muscle memory i need to remember to shake off